catalyst/registry.py

```diff
@@ -144,4 +144,43 @@ def _loggers_loader(r: registry.Registry):
 REGISTRY.late_add(_loggers_loader)
 
 
+def _torch_functional_loader(r: registry.Registry):
+    functions = [
+        "relu",
+        "hardtanh",
+        "hardswish",
+        "relu6",
+        "elu",
+        "selu",
+        "celu",
+        "leaky_relu",
+        "prelu",
+        "rrelu",
+        "glu",
+        "gelu",
+        "logsigmoid",
+        "hardshrink",
+        "tanhshrink",
+        "softsign",
+        "softplus",
+        "softmin",
+        "softmax",
+        "softshrink",
+        "gumbel_softmax",
+        "log_softmax",
+        "tanh",
+        "sigmoid",
+        "hardsigmoid",
+        "silu",
+        "normalize",
+    ]
+    import torch.nn.functional as F
+
+    for func in functions:
+        r[func] = getattr(F, func)
```

Comment:
could not we just add `F.__all__`? :)

Reply:
@elephantmipt let's use something like that - https://github.com/catalyst-team/catalyst/blob/master/catalyst/registry.py#L23#L24
what do you think?